test/ecs-appspec.test.ts

```diff
@@ -44,6 +45,11 @@ describe('CodeDeploy ECS AppSpec', () => {
     });
     const sg = ec2.SecurityGroup.fromSecurityGroupId(stack, 'sg', 'sg-00000000');
     const taskDefinition = ecs.FargateTaskDefinition.fromFargateTaskDefinitionArn(stack, 'taskdef', arn);
+    const testLambda = lambda.Function.fromFunctionArn(
```

Comment:
Rather than updating the existing test, please add a new test case for with hooks vs. without hooks

Reply:
Done